### DISCUSSION TITLE
In the new version of ggplot2, will a parameter be added to guide_colorbar, which can define to turn off the oneside ticks on the colobar?

### DISCUSSION THREAD

**Discussion by fengyufu** [ggplot2-extenders/ggplot-extension-club#4] [Ideas]
I am looking forward to the new version of `ggplot2` very much! ✨ Based on GitHub updates, I know there will be big changes in the `guides()` and some new parameters like `legend.ticks.length` will be added. So I'm curious if there will be some parameter to control the unilateral ticks switch that will increase the level of customization for the colorbar.

**Comment by teunbrand** [comment]
@Andtise do you mean like this?

``` r
library(ggplot2)
packageVersion("ggplot2")
#> [1] '3.5.0'

ggplot(mpg, aes(displ, hwy, colour = cty)) +
  geom_point() +
  theme(
    legend.ticks.length = unit(c(0.5, 0), "npc")
  )
```

![](https://i.imgur.com/zHMADM1.png)

<sup>Created on 2024-01-12 with [reprex v2.0.2](https://reprex.tidyverse.org)</sup>

**Reply by fengyufu** [reply]
Yes!!! Great teunbrand!😭😭  I can't wait to use the new version of ggplot2! Also, it should be able to set it to a negative number to make ticks face outward, right?

**Reply by teunbrand** [reply]
Yes, though you might want to adjust the relevant legend text margins too if the ticks are too long.
Also, using white outward facing ticks on a white background are going to be invisible.

**Reply by fengyufu** [reply]
That's awesome! I can't wait for it and express my heartfelt thanks for your work!!🥳
